executor/analyze_test.go

```diff
@@ -248,56 +245,35 @@ func (s *testSuite1) TestFastAnalyze(c *C) {
 	splitKeys := generateTableSplitKeyForInt(tid, []int{600, 1200, 1800, 2400})
 	manipulateCluster(cluster, splitKeys)
 
+	for i := 0; i < 3000; i++ {
```

Comment:
would it make UT run longer than 3 minutes?

Reply:
No, it just moves the origin code blocks here.